src/biom.hpp

```diff
@@ -66,11 +82,23 @@ namespace su {
             uint32_t **obs_indices_resident;
             double **obs_data_resident;
             unsigned int *obs_counts_resident;
+            
+            void malloc_resident(uint32_t n_obs);
 
+            /* allow load from scipy.sparse like data
+             * WARNING: assumes data are CSR, with observations as rows
```

Comment:
we should test the number of entries in `indptr` to make sure its shape makes sense to the shape of `id_`

Reply:
...we can't really do this, we *assume* `indptr` is of length n_obs + 1. This sanity check is the resposibility of the caller

Reply:
...but also this method isn't needed anymore